tests/upgrade_path/test_upgrade_path.py

```diff
@@ -28,6 +28,8 @@
 def pytest_generate_tests(metafunc):
     if "sad_case_type" in metafunc.fixturenames:
         sad_cases = SAD_CASE_LIST
+        if "multi_sad" in sad_cases:
```

Comment:
Please add a comment here for why this is removed.

Also, multi_sad should be removed only if other sub category cases are present in SAD_CASE_LIST (for completeness sake).

Reply:
@vaibhavhd I have added the comments, Can you help to merge it? Or @bingwang-ms  Can you help to merge? Thanks!